core/types/block.go

```diff
@@ -101,6 +101,9 @@ type Header struct {
 	// BlockGasCost was added by Apricot Phase 4 and is ignored in legacy
 	// headers.
 	BlockGasCost *big.Int `json:"blockGasCost" rlp:"optional"`
+
+	// ExcessDataGas was added by EIP-4844 and is ignored in legacy headers.
+	ExcessDataGas *big.Int `json:"excessDataGas" rlp:"optional"`
```

Comment:
Kept this to support EIP-4844 later, we don't need to take this now.

Reply:
I am removing this for now, we should review the full 4844 and check the verification before we take this change.

Reply:
Can we add verification into `plugin/evm/block_verification.go` as well?